Documentation changes (#121)
iot-lab/iot-lab-gateway: DOCKER.md

Docker Image of iot-lab-gateway
===============================

> The provided Dockerfile builds a Docker image that installs all  the necessary dependencies listed in [INSTALL.md](INSTALL.md), 

Reviewer: ```suggestion
The provided Dockerfile builds a Docker image that installs all the necessary dependencies listed in [INSTALL.md](INSTALL.md), 
```

> without you having to install them on the host machine. 

Reviewer: ```suggestion
without having to install them on the host machine.
```

> The only prerequisite on the host before running the container for the first time is installing udev rules on the host, which is done with:

Reviewer: ```suggestion
The only prerequisite on the host before running the container for the first time is to install udev rules on the host, which is done with:
```

    python setup.py udev_rules_install
    
Alternatively, 

>     cp bin/rules.d/* /etc/udev/rules.d/

Reviewer: ```suggestion
make setup-udev-rules
```

Does the same and is simpler.

Author: Agreed.
